### DISCUSSION TITLE
[Office Hours] Error when trying to fetch origin

### DISCUSSION THREAD

**Discussion by kc224** [CS342/.github#74] [Office Hours] [answered]
### Related Discussion

N/A

### Reproduction

I made my edits in a branch and then needed to fetch origin, which is creating an error (and therefore preventing me from submitting my pull request)

<img width="726" alt="Screenshot 2024-02-22 at 2 22 46 PM" src="https://github.com/CS342/.github/assets/63617639/b8c70929-3a72-4774-8783-dc6072cd3e40">
 

### Code of Conduct

- [X] I agree to follow this course's Code of Conduct and Contributing Guidelines

**Comment by bauer-andreas** [comment, marked as answer]
To me it seems like you placed your repository within your iCloud Drive directory. Generally, I would not recommend placing your project folders within your iCloud Drive directory (e.g., when you have iCloud Drive turned on for your Documents folder).

`refs/heads/migrated-onboarding 2` seems like a duplicate created by iCloud Drive during synchronization (It is located within the hidden `.git` folder within your project root). I would recommend to just delete that file. Important, do **not** delegate `refs/head/migrated-onboarding` but only the `migrated-onboarding 2` file in the `.git/refs/heads` directory (with space and the digit 2 at the end).

Hope that helps.

**Comment by kc224** [comment]
That worked, thank you so much! Going forward, should I just turn iCloud Drive off for my Nourish folder within my documents folder?

**Reply by PSchmiedmayer** [reply]
I would generally avoid to work with a GitRepo in iCloud Drive, this is only causing issues. I would suggest move it to a folder outside of iCloud Drive.
